### ISSUE TITLE
Feature Request: load tinymce from assets..

### ISSUE THREAD

**Issue by bluscreen** [marucjmar/ember-cli-tinymce#24]
Hi marucjmar

I would very much prefer to load the tinymce locally instead of cdn.tinymce.com
Do you think you can integrate this feature?

Cheers
Daniel

**Comment by marucjmar** [comment]
Hi @bluscreen 

Good idea 👍  
Tomorrow I'll take care of it

**Comment by bluscreen** [comment]
Hi marucjmar

any progress on this? otherwise I guess I'll have to manually install tinymce via node  :face_with_head_bandage:

**Comment by bluscreen** [comment]
in case youre still interested, i got an approach running now using npm and ember browserify. its still quite brittle but it works..

**Comment by marucjmar** [comment]
I'm working on it. You can see my progress on branch `load-from-assets`. In dev environment work correctly, but not when build production app.

**Comment by bluscreen** [comment]
Hi thats probably due to minification and or fingerprinting. You can
disable this in ember cli build config. Cheers Daniel

Marcin Lazar <notifications@github.com> schrieb am Mo., 26. März 2018,
09:32:

**Comment by ziali088** [comment]
++ also waiting for this

**Comment by ziali088** [comment]
Hi @marucjmar,

Could you roll this out? ember-cli-tinymce is always using tinymce version 4.4 for me and that has a bug with image attachments that's affecting me and is fixed in later versions ([link to issue](https://github.com/tinymce/tinymce/issues/3611).

Thanks for the contribution!

**Comment by marucjmar** [comment]
@ziali088 can you show your environment.js config? 
this solution does not help?

```
   tinyMCE: {
      load: true,
      version: 4.5
    }
```

**Comment by ziali088** [comment]
Yes updating the config does fix it, but my concern was more of a now buggy version being the default :)
